src/Circle.js

```diff
@@ -51,6 +53,13 @@ function getPathStyles(offset, percent, strokeColor, strokeWidth, gapDegree = 0,
 class Circle extends Component {
   paths = {};
 
+  gradientId = 0;
+
+  constructor() {
+    super();
+    this.gradientId = gradientSeed++;
```

Comment:
`gradientSeed += 1;` instead.

Newer rc has eslint rule of `no-plusplus`.

Reply:
Shall I add a .eslint file?

Reply:
We use `father` as package builder for new rc-component. This can be done in another PR. Let keep this PR simple.

Reply:
Got ya.